app/services/email_checker.rb

```diff
@@ -83,4 +86,8 @@ def mx_records?
       Rails.configuration.mx_checker.records?(domain)
     end
   end
+
+  def sendgrid_api
+    @sendgrid_api ||= SendgridApi.new
```

Comment:
I'm not sure how we've got puma configured (I don't think we're using forking, but I may be wrong...). Can you investigate?